source/docs/software/wpilib-tools/pathweaver/integrating-robot-program.rst

```diff
@@ -37,3 +37,5 @@ The ``fromPathweaverJson`` (Java) / ``FromPathweaverJson`` (C++) static methods
        frc::Trajectory trajectory = frc::TrajectoryUtil::FromPathweaverJson(deployDirectory);
 
 .. note:: In the examples above, ``YourPath`` should be replaced with the name of your path.
+
+.. warning:: Loading a PathWeaver JSON from file can take up to 7 seconds per file so it is highly recommended that the robot load these paths on startup.
```

Comment:
I'm not sure that this is true on C++ as discussed in the allwpilib issue.

Reply:
It can be reworded to say that it's a blocking or lengthy operation without specifying time.

Reply:
I know that running in Sim isn't comparable.  I can't say if it is present on a C++ robot so, what do you want me to do unless a C++ person is able to test?